erpnext/stock/get_item_details.py

```diff
@@ -165,6 +165,7 @@ def get_basic_details(args, item):
 		"discount_percentage": 0.0,
 		"supplier": item.default_supplier,
 		"delivered_by_supplier": item.delivered_by_supplier,
+		"actual_qty": get_available_qty(item.name, user_default_warehouse or args.warehouse or item.default_warehouse)
```

Comment:
Not needed, it already maps actual qty via https://github.com/frappe/erpnext/blob/develop/erpnext/stock/get_item_details.py#L46